notebooks/text_models/solutions/classify_text_with_bert.ipynb

```diff
@@ -9,9 +9,9 @@
     "# Classify text with BERT\n",
```

Comment:
<p>"The text pre-processor is a TensorFlow model." -&gt; "The text pre-processor is a subclass of <code>Keras.layers.Layer</code> object"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/660/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>